dashboard/src/shared/App.ts

```diff
@@ -19,11 +19,12 @@ export class App {
   public static async create(
     releaseName: string,
     namespace: string,
+    kubeappsNamespace: string,
     chartVersion: IChartVersion,
     values?: string,
   ) {
     const chartAttrs = chartVersion.relationships.chart.data;
-    const repo = await AppRepository.get(chartAttrs.repo.name);
+    const repo = await AppRepository.get(chartAttrs.repo.name, kubeappsNamespace);
     const auth = repo.spec.auth;
```

Comment:
Not from this patch but just pointing out inconsistent usage of es6 destructuring.

const { auth } = repo.spec;

Reply:
Should I fix this in this PR?

Reply:
No